src/cli/commands/channel.rs

```diff
@@ -0,0 +1,26 @@
+use clap::Subcommand;
+
+#[derive(Subcommand)]
+pub(crate) enum Commands {
+    Add {
+        name: String,
+
+        app_id: String,
+
+        #[clap(short, long)]
+        domain: Option<String>,
+
+        #[clap(long)]
+        range_rule: Option<String>,
+
+        #[clap(long)]
+        revision_id: Option<String>,
+
+        #[clap(long)]
+        certificate_id: Option<String>,
+    },
+    Remove {
```

Comment:
Creating a channel works, but removing it returns a 404:

```
❯ hippo channel add staging 2f0c046f-ad29-4846-892b-b850e7d0eba7
Added Channel staging (ID = 'e5114347-fba8-4d19-93c0-59a4cd3e54b4')
IMPORTANT: save this Channel ID for later - you will need it to update and/or delete the Channel (for now)

❯ hippo channel remove e5114347-fba8-4d19-93c0-59a4cd3e54b4
Error: error in response: status code 404 Not Found
```

Reply:
https://github.com/deislabs/hippo-cli/pull/99#discussion_r794740312